w3c.json

```
@@ -0,0 +1,5 @@
+{
+    "group": "cg/privacycg",
+    "contacts": ["hober", "TanviHacks", "erik-anderson", "weiler"],
```

```suggestion
    "contacts": ["hober", "martinthomson", "erik-anderson", "weiler"],
```

We should update this everywhere :)